Trivago/Jade/Application/JsonApi/Request/RequestFactory.php

```diff
@@ -301,14 +310,18 @@ private function parseFilters(ResourceConfig $resourceConfig, CompositeFilter $c
     /**
      * Validating the "path" is not under "excluded_attributes" config section
      * @param ResourceConfig $resourceConfig
-     * @param $key
-     * @param $path
+     * @param string $key
+     * @param string $path
      */
     private function validateExcludedPaths(ResourceConfig $resourceConfig, $key, $path)
     {
         if ($this->strictFilteringAndSorting && $resourceConfig->isAttributeExcluded($resourceConfig->getRealPath($path))) {
-            throw InvalidRequest::createWithMessage($key, 'invalid_path', sprintf('The path "%s" is a non rendered column. 
+            if ($this->jsonApiDebug) {
+                throw InvalidRequest::createWithMessage($key, 'invalid_path', sprintf('The path "%s" is a not rendered column. 
                 Try to remove this property from the "excluded_attributes" config section or set "strict_filtering_and_sorting" to false', $path));
+            } else {
+                throw InvalidRequest::createWithMessage($key, 'invalid_path', 'Some excluded column path have been used to filter or sort');
```

Comment:
Well you're still exposing the fact that there is such a column but it's excluded 😄 
Use the same message as when it doesn't exists (There is no path called $path)